pages/content/amp-dev/community/roadmap.html

```diff
@@ -2,86 +2,185 @@
-{% do doc.amp_dependencies.add('amp-bind', '0.1') %}
 
 <main class="ap--main ap-t-roadmap">
-  <div class="ap--container">
-    <div class="ap-o-roadmap">
-
-      <amp-state id="roadmapfilter">
-        <script type="application/json">
-          {
-            "activefilter": "all"
-          }
-        </script>
-      </amp-state>
-
-      {% do doc.styles.addCssFile('css/components/organisms/filter-bubbles-list.css') %}
-
-      <section class="ap--content">
-        <h1>{{ _('The AMP Project Roadmap') }}</h1>
-        <div class="ap-m-tip ap-m-tip-note">
-          <div class="ap-m-tip-content">
-            {% do doc.styles.addCssFile('css/components/molecules/tip.css') %}
-            <p>{{ _('This is a high level overview suitable for all audiences. For a more detailed developer view, <a href="https://github.com/ampproject/amphtml/projects/43">head to GitHub.</a>') }}</p>
-          </div>
+  <section class="ap--stage ap--container-fluid">
+    {% do doc.styles.addCssFile('css/components/organisms/stage.css') %}
+
+    <div class="ap-o-stage">
+      <div class="ap--container-fluid">
+        <div class="ap-o-stage-content">
+          <h1 class="ap-o-stage-content-headline">{{ _('The AMP Project Roadmap') }}</h1>
+        </div>
+        <div class="ap-o-stage-image">
+          {% do doc.icons.useIcon('icons/roadmap-header.svg') %}
+          <svg style="height: 80%; width: 100%;"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#roadmap-header"></use></svg>
         </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="ap-t-roadmap-intro">
+    <p class="ap-t-roadmap-intro-text">An AMP Working Group is a segment of the community with knowledge/interest in specific area of AMP. Working Groups are created by AMP’s Technical Steering Committee.</p>
+    <div class="ap-m-tip ap-m-tip-info">
+      <div class="ap-m-tip-content">
+        {% do doc.styles.addCssFile('css/components/molecules/tip.css') %}
+        <p>{{ _('This is a high level overview, to look at detailed features up for consideration please head to <a href="https://github.com/ampproject/amphtml/projects/43">Github</a>. You can vote on issues, leave feedback or create a new issue.') }}</p>
+      </div>
+    </div>
+  </section>
 
+  <section class="ap-t-roadmap-filter" id="status-updates">
+    {% do doc.styles.addCssFile('css/components/organisms/filter-bubbles-list.css') %}
+    {% do doc.icons.useIcon('icons/close.svg') %}
 
-        <!-- Filter bubbles -->
-        {% do doc.icons.useIcon('icons/close.svg') %}
-        <div class="ap-o-roadmap-filter-list all" [class]="roadmapfilter.activefilter != 'all' ? 'ap-o-roadmap-filter-list filtered' : 'ap-o-roadmap-filter-list all' ">
-          {% for label in doc.roadmap.labels %}
+    <h2 class="ap-t-roadmap-filter-headline">{{ _('Select working groups') }}</h2>
+    <div class="ap-t-roadmap-filter-list">
+      <button class="ap-m-filter-bubble reset [% if not category %]active[% endif %]"
```

Comment:
Please make this (and the other `button`s) actual `a`s instead of a `button` with `AMP.navigateTo`. This helps UX as for example you are then able to right-click and copy links to a certain working group.